apps/meteor/tests/e2e/omnichannel/omnichannel-units.spec.ts

```diff
@@ -138,29 +141,75 @@ test.describe('OC - Manage Units', () => {
 			await poOmnichannelUnits.findRowByName(editedUnitName).click();
 			await expect(poOmnichannelUnits.contextualBar).toBeVisible();
 
-			await test.step('expect confirm delete unit', async () => {
-				await test.step('expect to be able to cancel delete', async () => {
-					await poOmnichannelUnits.btnDelete.click();
-					await expect(poOmnichannelUnits.confirmDeleteModal).toBeVisible();
-					await poOmnichannelUnits.btnCancelDeleteModal.click();
-					await expect(poOmnichannelUnits.confirmDeleteModal).not.toBeVisible();
-				});
+			await test.step('expect to confirm delete', async () => {
```

Comment:
these amount of stuff is unnecessary, left just the test on the list of units after deleting it